include/RAJA/Layout.hxx

```diff
@@ -55,9 +55,9 @@
 
 #include <iostream>
 #include <limits>
-#include "RAJA/IndexValue.hxx"
-#include "RAJA/LegacyCompatibility.hxx"
-#include "RAJA/foralln/Permutations.hxx"
+#include "RAJA/internal/IndexValue.hxx"
+#include "RAJA/internal/LegacyCompatibility.hxx"
+#include "RAJA/internal/foralln/Permutations.hxx"
```

Comment:
See my reply above to Adam - I just moved everything to internal and then have been pulling stuff out. The IndexValue stuff should move to the top level.